src/test/ui/pattern/usefulness/top-level-alternation.rs

```diff
@@ -1,7 +1,7 @@
-#![feature(let_else)]
-
 #![deny(unreachable_patterns)]
 
+
+
```

Comment:
I'm assuming the blank lines here and in various other tests are to avoid having to re-bless the tests?

That's fine for temporary purposes, but the final PR should go ahead and re-bless the tests rather than workign around it like this.

Reply:
> I'm assuming the blank lines here and in various other tests are to avoid having to re-bless the tests?

Yeah I didn't want to cause stderr churn. It's been suggested for an [earlier PR](https://github.com/rust-lang/rust/pull/88316#issuecomment-905796963) of mine that also touched the testsuite so I did the same here. If it's fine that I remove those lines, I'll do that instead.